### DISCUSSION TITLE
can not login to apple store sonoma

### DISCUSSION THREAD

**Discussion by rubinglen** [quickemu-project/quickemu#1471] [Quickemu]
I am trying to install an app from the appstore.  There is a page that comes up wanting to confirms my identity most of it is pre-filled.  I fill in what is missing the 4 digit cv code and the telephone and area code.  When i press continue it insists that I have not filled in the cv code or phone number and i need to do so in order to continue.  I have tried logging in separately on my other machine and agree to any terms, but regardless I am prompted for this on emu machine and even though it looks like the correct fields are filled they are not detected upon form submission.

**Comment by daveschz** [comment]
Did you follow this step in the Wiki?
https://github.com/quickemu-project/quickemu/wiki/03-Create-macOS-virtual-machines#macos-app-store
I just installed and had no trouble after making sure my ethernet was set to en0

**Comment by rubinglen** [comment]
i was only able to fix this by registering with the itunes store on another
apple device.  i wasn't registered at all with app store, so that was the
problem.

**Comment by lucascompython** [comment]
Hey there, I installed macOS-Sonoma today, and it actually came with Sequoia, I don't know if that's an issue, but I can't log in to my Apple account, I get this error when trying to log in on the app store: 
![image](https://github.com/user-attachments/assets/65a0a2cf-b83b-49a8-ac99-8f0f9ba55f7f)
And this error when trying to log in in the settings app:
![image](https://github.com/user-attachments/assets/a8d2d93a-6485-470a-b07b-ed794cf5c6e4)
I've tried following the wiki and deleting the following services and running the command stated in the wiki, but it still did not work.
![image](https://github.com/user-attachments/assets/66dcf7a5-5e2e-4a0a-994c-15645751f74f)
They keep reappearing after restarting, I don't know if that's supposed to happen.

**Reply by eLindros** [reply]
I did the same with the same result. Could you resolve it by any chance?

**Reply by lucascompython** [reply]
It still doesn't work for me. I've updated to 15.3.1 and no luck.

**Reply by baileydel** [reply]
This is because macOS has implemented VM detection, here is a guide that bypasses the vm detection
https://forum.proxmox.com/threads/anyone-can-make-bluetooth-work-on-sonoma.153301/#post-697832
from fty1777

Basically you download OCAuxiliaryTools inside of the mac, run it, ignore the security issues, and apply the patches.

I would also recommend following this tutorial to update opencore https://www.youtube.com/watch?v=S6mE44a-Z-8

The first time i tried patching macOS it didn't work until i updated openCore

**Comment by Noah765** [comment]
I have documented a workaround [here](https://github.com/quickemu-project/quickemu/issues/1547#issuecomment-2708376222).
